programs/dynamic-bonding-curve/src/instructions/migration/dynamic_amm_v2/migrate_damm_v2_initialize_pool.rs

```diff
@@ -310,6 +297,21 @@ impl<'info> MigrateDammV2Ctx<'info> {
             )?;
         }
 
+        let after_lamports = self.pool_authority.lamports();
+        let consumed_lamports = before_lamports.safe_sub(after_lamports)?;
```

Comment:
can use saturating_sub? in case weird thing happen from `third party`

Reply:
Fixed